tests/DynatraceMetricExporter.spec.ts

```diff
@@ -123,6 +123,9 @@ describe("MetricExporter.export", () => {
 								startTime: [0, 0],
 								value: {
 									sum: 22.4,
+									hasMinMax: extrema != null,
+									min: extrema?.min ?? Infinity,
+									max: extrema?.max ?? -1,
```

Comment:
Should probably be -Inf, as the Histogram can also contain negative values (or is that impossible in JS?)

Reply:
Spec says negative is not allowed IIRC but the JS sdk currently doesn't check.

Reply:
We drop values that are `<0` on the histogram instrument. However, I just realized that we're not checking that on, for example, UpDownCounters. I think with the current implementation, it will still work with `max = -1`, but I'm checking that right now to make sure. That being said, we should probably change the default value to `-Infinity` in the JS SDK as well to avoid confusion.

Reply:
I am surprised, I am sure that exponential histograms can have negative values, and I thought explicit bucket histograms could as well. I did a quick pass on the spec and did not find any mention of negative values being disallowed. Do you have a link to the spec where it disallows these?

Reply:
@pirgeo this line [here](https://github.com/open-telemetry/opentelemetry-specification/blob/94ab1fe580f7e00926bd8e627d8924400a0ecc5f/specification/metrics/api.md?plain=1#L698) mentions that. :slightly_smiling_face: 

Here's the issue I created on OTel JS: https://github.com/open-telemetry/opentelemetry-js/issues/3085
And here's the fix for that: https://github.com/open-telemetry/opentelemetry-js/issues/3086

Reply:
I must have missed that - thank you!